system_stats.c

```diff
@@ -178,4 +235,12 @@ void system_stats_init(void)
 	struct utsname un;
 	if (uname(&un) == 0)
 		sysname = pstrdup(un.release);
+
+	if (memory_cgroup_mount != NULL) {
+		const char prefix[] = "/memory.";
+		memory_cgroup_len = strlen(memory_cgroup_mount);
+		memory_cgroup = repalloc(memory_cgroup_mount, memory_cgroup_len + 23);
```

Comment:
Yeah, obviously, but the point is that it deserves a code comment.

It would also help to reflect the fact that `memory_cgroup` is a filename (as opposed to `cgroup_memory`).  Otherwise it's pretty confusing.